mava/wrappers/jaxmarl.py

```diff
@@ -297,23 +281,55 @@ def discount_spec(self) -> specs.BoundedArray:
 
     def action_mask(self, state: JaxMarlState) -> Array:
         """Get action mask for each agent."""
-        if self.has_action_mask:
-            avail_actions = self._env.get_avail_actions(state)
-            mask = jnp.array(batchify(avail_actions, self.agents), dtype=jnp.float32)
-        else:
-            mask = jnp.ones(self._action_shape, dtype=jnp.float32)
-        return mask
+        pass
+
+    def get_global_state(self, env_state: Array, obs: Dict[str, Array]) -> Array:
+        """Get global state from observation for each agent."""
+        pass
```

Comment:
make this and action_mask abstract methods (also I just prefer `...` over `pass`, but that's up to you :smile: )

```suggestion
    @abstractmethod
    def get_global_state(self, env_state: Array, obs: Dict[str, Array]) -> Array:
        """Get global state from observation for each agent."""
        ...
```

Reply:
Done 😅